script/1_0_2current_en_script/Ciel/Day 1/01_01_CIEL01_3_3_2.txt

```diff
@@ -113,7 +113,7 @@ Trying to light up the mood, she forces a smile.
 "But it was a little unexpected. I thought you were a laid-back kid."
 <Page33>
 "Yeah, I'm fundamentally a pretty relaxed person. I've made it my motto to enjoy the present as much as possible.
-Isn't it better to look forward to what's ahead than what's already over?"
+Isn't it better to look forward to what's ahead than to what's already over?"
```

Comment:
I feel like generally you'd contrast looking forward with looking back, so perhaps
> better to look forward to what's ahead than back on what's already over?